public/data/Arushay.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Arushay",
+  "type": "personal",
+  "bio": "React Developer || Open Source Enthusiast",
+  "avatar": "https://github.com/Arushay.png",
+  "links": [
+    {
+      "name": "Blog",
+      "url": "https://aymandin.hashnode.dev",
+      "icon": "book"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/EmanDin1",
+      "icon": "twitter"
+    },
+    {
+      "name": "Check out my portfolio",
+      "url": "https://arosha-portfolio.netlify.app/#about",
```

Your link is directly heading towards the `About` section of your portfolio instead of the `hero` section.

If you wanted to remove it!?

---

**Summary:** I have a question ❓
```suggestion
      "url": "https://arosha-portfolio.netlify.app/",
```